MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableProtocol.c

```diff
@@ -1995,6 +2098,23 @@ AcpiTableAcpiTableConstructor (
     SdtAcpiTableAcpiSdtConstructor (AcpiTableInstance);
   }
 
+  //
+  // Check FSP ACPI Hob.
+  //
+  GuidHob = GetFirstGuidHob (&gAcpiTableHobGuid);
+  if (GuidHob != NULL) {
+    Status = InstallAcpiTableFromAcpiSiliconHob (AcpiTableInstance, GuidHob);
+    if (Status == EFI_SUCCESS) {
+      DEBUG ((DEBUG_INFO, "Installed ACPI Table from AcpiSiliconHob.\n"));
+      return EFI_SUCCESS;
+    } else {
+      DEBUG ((DEBUG_ERROR, "Fail to Installed ACPI Table from AcpiSiliconHob!!\n"));
+      ASSERT (Status != EFI_SUCCESS);
+    }
+  } else {
+    DEBUG ((DEBUG_ERROR, "Fail to locate AcpiSiliconHob!!\n"));

```

Comment:
Here DEBUG level should be DEBUG_INFO intead of ERROR. Some platform boot doesn't depend on FSP.